setup.py

```diff
@@ -92,8 +92,8 @@ def write_git_revision(package_name):
           "pytools>=2018.4",
           "pymbolic>=2019.1",
           "genpy>=2016.1.2",
-          "cgen>=2016.1",
-          "islpy>=2019.1",
+          "cgen>=2016.1,<2019.1",
+          "islpy>=2016.2",
```

Comment:
Can we keep the `islpy` requirement aligned with upstream?

Reply:
Fixed!